src/components/icon-tile/icon-tile.module.css

```diff
@@ -59,87 +59,91 @@ but is necessary for us to implement the current designs for the
 /sentinel and /hcp product landing pages.
 */
 .color-neutral-dark {
-  --icon-color: var(--token-color-foreground-high-contrast);
-  --border-color: var(--token-color-foreground-strong);
-  --background: var(--token-color-foreground-primary);
+	--icon-color: var(--token-color-foreground-high-contrast);
+	--border-color: var(--token-color-foreground-strong);
+	--background: linear-gradient(
+		315deg,
+		var(--token-color-palette-neutral-700) 0%,
+		var(--token-color-palette-neutral-600) 100%
+	);
```

Comment:
This is the meaning change to this file, beyond formatting.

Reply:
https://github.com/hashicorp/dev-portal/pull/975/files?w=1

Reply:
ahh the whitespace trick! ty I forgot the shorthand